frame/collective/src/lib.rs

```diff
@@ -760,7 +779,10 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			}
 		} else {
 			if position_no.is_none() {
-				voting.nays.push(who.clone());
+				voting
+					.nays
+					.try_push(who.clone())
+					.expect("Proposal voting nays can't overflow; qed");
```

Comment:
Runtime upgrade can change constants like `MaxMembers` and then these invariants may not hold anymore.